### ISSUE TITLE
메인 페이지 HTML 작업 중...

### ISSUE THREAD

**Issue by demian0721** [Jongveloper/hanghae2toyproject1#1]
<!DOCTYPE html>
<html lang="en">
<head>
    
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
          integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">

    
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>

    <title>항해 2기 사전 스터디 2조 토이프로젝트</title>

    <style>
        .wrap {
            width: 900px;
            margin: auto;
        }

        .logoline {
            width: 900px;
            margin-bottom: 50px;
        }

        .logoimg {
            width: 150px;
        }

        .title {
            font-size: 20px;
            margin-left: 150px;
        }

        .btn-login-out {
            margin-left: 150px;
            font-size: 14px;
        }

        .menu {
            width: 880px;
            margin: auto;
            margin-bottom: 50px;
        }

        .contents {
            width: 850px;
            margin: auto;
        }

        .guide {
            background-color: skyblue;
            text-align: center;

            width: 300px;
            height: 200px;

            padding-top: 20px;
            margin-bottom: 30px;
        }

        .bloglink {
            background-color: darksalmon;
            text-align: center;

            width: 300px;
            height: 200px;

            padding-top: 5px;
        }
        .calendar {
            background-color: green;
            text-align: center;

            width: 550px;
        }
    </style>
</head>
<body>
<div class="wrap">
    <div class="top">
        <div class="logoline">
            <a href="https://www.notion.so/hh99team2/2-2-182466e1abed4cc98415d20536ad490c"><img class="logoimg"
                                                                                                src="https://blog.kakaocdn.net/dn/4ngLN/btq4t4EuUo4/qTfDlHMnessbdCSwkugkFk/img.jpg"></a>
            <span class="title">항해 2기 사전 스터디 2조 토이프로젝트
        <button type="button" class="btn btn-primary signup-btn btn-login-out">로그아웃</button>
        </div>
        <div class="menu">
            <ul class="nav nav-tabs">
                <li class="nav-item">
                    <a class="nav-link active" aria-current="page" href="#">HOME</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#">우종혁</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#">이선민</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#">주재인</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#">김나영</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#">공부방</a>
                </li>
                
                
                
            </ul>
        </div>
    </div>
    <div class="contents">
        <table class="bodytable">
            <tr>
                <td class="guide">
                    <p>사전스터디 첫 팀 프로젝트!<br>
                        자기소개 페이지 및 공부방 사이트입니다.<br>
                        각자의 취미 및 관심사, 맛집 등의 정보와 그동안 공부한 것들을 기록 했습니다.</p>
                </td>
                <td class="calendar" rowspan="2">
                    캘린더 들어갈 자리
                </td>
            </tr>
            <tr>
                <td class="bloglink">
                    <p>&gt;&gt;개인 개발 블로그 링크&lt;&lt;</p>
                    <p><a href="https://destory.tistory.com">우종혁</a></p>
                    <p><a href="https://velog.io/@sunm309">이선민</a></p>
                    <p><a href="https://axis.tistory.com/">주재인</a></p>
                    <p><a href="https://velog.io/@skdud4659">김나영</a></p>
                </td>
            </tr>
        </table>
    </div>
</div>

</body>
</html>
